### PR TITLE
Fixed storage tests

### PR DESCRIPTION
**Pull request by VladiMihaylenko**

### CODE REVIEW

**Comment by bykoianko** [comment]
DUDE JTTP

File: storage/storage_tests/storage_tests.cpp [hunk 1]
```diff
@@ -844,29 +844,26 @@ UNIT_CLASS_TEST(TwoComponentStorageTest, DownloadTwoCountriesAndDelete)
...
+                        Status::ENotDownloaded});
     uruguayChecker->StartDownload();
     venezuelaChecker->StartDownload();
     storage.DeleteCountry(uruguayCountryId, MapOptions::Map);
     storage.DeleteCountry(venezuelaCountryId, MapOptions::CarRouting);
     runner.Run();
   }
-  // @TODO(bykoianko) This test changed its behaivier. This commented lines are left specially
+  // @TODO(bykoianko) This test changed its behavior. This commented lines are left specially
   // to fixed later.
```
Comment (bykoianko) [inline storage/storage_tests/storage_tests.cpp]: Давай удалим эту todo

File: storage/storage_tests/storage_tests.cpp [hunk 2]
```diff
@@ -844,29 +844,25 @@ UNIT_CLASS_TEST(TwoComponentStorageTest, DownloadTwoCountriesAndDelete)
     unique_ptr<CountryDownloaderChecker> uruguayChecker = make_unique<CountryDownloaderChecker>(
         storage, uruguayCountryId, MapOptions::MapWithCarRouting,
         vector<Status>{Status::ENotDownloaded, Status::EDownloading, Status::ENotDownloaded});
-    // Only routing file will be deleted for Venezuela, thus, Venezuela should pass through
-    // following
-    // states:
-    // NotDownloaded -> InQueue (Venezuela is added after Uruguay) -> Downloading -> Downloading
-    // (second notification will be sent after deletion of a routing file) -> OnDisk.
+    // Despite only routing file will be deleted for Venezuela, the map file should also be deleted and
```
Comment (syershov) [inline storage/storage_tests/storage_tests.cpp]: The comment is incorrect, we no longer have any routing files.

**Comment by VladiMihaylenko** [comment]
@syershov ptal

**Pull request merged**